index.html

```diff
@@ -107,9 +113,26 @@ <h2 class="head-description1">Profesional Art Printing Data</h2>
                 See Project
             </button>
         </section>
-        <section class="content1">
+        <section class="content2">
             <div>
-                <h2 class="head-description1">Profesional Art Printing Data</h2>
+                <h2 class="head-description1">Data Dashboard Healthcare</h2>
+                <p class="project-description1">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required. Has 
+                    been the industry's standard.</p>
+                <ul class="programming-language">
+                    <li class="item1">CSS</li>
+                    <li class="item1">HTML</li>
+                    <li class="item1">Ruby</li>
+                    <li class="item1">Bootstrap</li>
+                </ul>
+            </div>
+            <button class="button-project1">
+                See Project
+            </button>
+        </section>
+        <section class="content3">
+            <div>
+                <h2 class="head-description1">Website Portfolio </h2>
```

Comment:
Kindly reduce the image height here to have a better UI of your project display

Figma design :
![image](https://github.com/AndresEspi/portafolio/assets/20537712/8f24ec8a-f628-4541-b3ec-730697c95312)

Your design :
![Uploading image.png…]()